### PR TITLE
Added navbar and header

### PR DESCRIPTION
**Pull request by Bhavik-Ardeshna**
#9 #7 
Screenshot from 2021-03-12 13-57-46

### CODE REVIEW

**Comment by srimani-programmer** [comment]
@Bhavik-Ardeshna Did you make this PR AS part of GSSOC?

**Comment by Bhavik-Ardeshna** [comment]
yes

**Comment by Bhavik-Ardeshna** [comment]
I think this PR is ready to merge please review :)

File: implementation.py [hunk 1]
```diff
@@ -6,48 +6,51 @@
...
+        X, y, test_size=0.2, random_state=0)
 
+    # Feature Scaling
+    from sklearn.preprocessing import StandardScaler
+    global sc
+    sc = StandardScaler()
+    X_train = sc.fit_transform(X_train)
+    X_test = sc.transform(X_test)
 
-	# Feature Scaling
```
Comment (srimani-programmer) [inline implementation.py:27]: @Bhavik-Ardeshna Why there is an unnecessary spaces although the file

File: random_forest.py [hunk 2]
```diff
@@ -2,39 +2,40 @@
...
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelEncoder, OneHotEncoder
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 
 # Importing the dataset
-dataset = pd.read_csv('Breast Cancer Data.csv')
```
Comment (srimani-programmer) [inline random_forest.py:10]: @Bhavik-Ardeshna why you are trying to change the filename.

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]
@Bhavik-Ardeshna there are a lot of spam changes in the PR. please don't do that. If you want to contribute to this project please read the guidelines properly.

**Comment by srimanikantapalakollu** [comment]
@Bhavik-Ardeshna Your code is conflicting with the project code base. please read the guidelines properly and make new PR.

**Pull request closed without merging**